src/quantum/README.rst

```diff
@@ -0,0 +1,188 @@
+
+   .. note::
+      In case you have more than one subscription associated with your Azure account you must specify the 
+      subscription you want to use. You can do this with the command `az account set -s <Your subscription ID>`.
+
+
+2. Install the Quantum extension for the Azure CLI
+
+   .. code-block::
+
+      az extension add --name quantum
+
+
+3. You can see all the Azure Quantum workspaces in your subscription with the `az quantum workspace list` command:
+
+   .. code-block::
+
+      az quantum workspace list
+
+
+4. You can use `quantum workspace set` to select a default workspace you want to use 
+   to list and submit jobs. Note that you also need to specify the resource group.
+
+   .. code-block::
+
+      az quantum workspace set -g MyResourceGroup -w MyWorkspace  -o table
+
+      Location     Name                               ResourceGroup
+      -----------  ---------------------------------  --------------------------------
+      westus       ws-yyyyyy                          rg-yyyyyyyyy
+
+
+5. You can check the current default workspace with command `az quantum workspace show`.
+
+   .. code-block::
+
+      az quantum workspace show -o table
+
+      Location     Name                               ResourceGroup
+      -----------  ---------------------------------  --------------------------------
+      westus       ws-yyyyyy                          rg-yyyyyyyyy
+
+
+6. You can see all the jobs submitted to a workspace using `az quantum job list`.
+
+   .. code-block::
+
+      az quantum job list -o table
+
+      Id                                    State    Target          Submission time
+      ------------------------------------  -------  --------------  ---------------------------------
+      yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy  Waiting  MyProvider.MyTarget  2020-06-12T14:20:18.6109317+00:00
+
+   The console will output the information about the job, including the ID of the job.
+
+
+7. You can use the ID of the job to track its status.
+
+   .. code-block::
+
+      az quantum job show -id yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy -o table
+
+      Id                                    State    Target          Submission time
+      ------------------------------------  -------  --------------  ---------------------------------
+      yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy  Waiting  MyProvider.MyTarget  2020-06-12T14:20:18.6109317+00:00
+
+
+8. Once the job finishes you can visualize the job's results with `az quantum job output`:
+
+   .. code-block::
+
+      az quantum job output -id yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy -o table
+   
+      Result    Frequency
+      --------  -----------  -------------------------
+      [0,0]     0.25000000   ▐█████                  |
+      [1,0]     0.25000000   ▐█████                  |
+      [0,1]     0.25000000   ▐█████                  |
+      [1,1]     0.25000000   ▐█████                  |
+
+
+Submitting Q# programs for execution from the command line
```

Comment:
Can we merge this section with above content. They looks like two independent articles.

Reply:
Done. :)